src/middlewared/middlewared/etc_files/ftpusers.mako

```diff
@@ -0,0 +1,8 @@
+<%
+    ftp = render_ctx['ftp.config']
+    users = [user["username"] for user in render_ctx['user.query']]
+
+    # Exclude root
+    ftpusers = [user for user in users if user != 'root']
+%>
+${"\n".join(ftpusers) + "\n"}
```

Comment:
Let's add a comment to this file that it defines users who are denied FTP access and make sure it contains all `builtin` users excluding `ftp`.